packages/expo-analytics-amplitude/src/Amplitude.ts

```diff
@@ -23,56 +23,56 @@ export interface AmplitudeTrackingOptions {
   disableVersionName?: boolean;
 }
 
-export function initialize(apiKey: string): Promise<void> {
+export function initialize(apiKey: string) {
   if (!ExpoAmplitude.initialize) {
     throw new UnavailabilityError('Amplitude', 'initialize');
   }
-  return ExpoAmplitude.initialize(apiKey);
+  ExpoAmplitude.initialize(apiKey);
```

Comment:
Why wouldn't we want to return a Promise for the users to be able to await these calls? I imagine one could want to await for Amplitude to be initialized or `setUserId before logging events?

`ExpoAmplitude.initialize`, even though it doesn't say so, as a unimodule method, exposes only asynchronous methods, so the result of this call will always be a Promise.

Reply:
Ah yeah I see what you're saying. In that case I'll just rename all the methods to suffix them with Async and return the promise from the UM method